independent-projects/qute/core/src/main/java/io/quarkus/qute/CompletedStage.java

```diff
@@ -21,12 +21,19 @@
 public final class CompletedStage<T> implements CompletionStage<T>, Supplier<T> {
 
     static final CompletedStage<Void> VOID = new CompletedStage<>(null, null);
+    static final CompletedStage<Object> NULL = new CompletedStage<>(null, null);
```

Comment:
How different is this from the `VOID` above?

Reply:
It's `CompletedStage<Void>` vs `CompletedStage<Object>` but you're right that we can reuse the same object and cast it when necessary... will update the PR. Thanks!